HLTrigger/Tools/scripts/timingPdfMaker.py

```diff
@@ -36,443 +36,443 @@
-                if 'pathTime_' in allnames:
-                        pathname = '_'.join(allnames.split('_')[1:])
-                        if not pathname in names1:
-                                names1[pathname] = k.ReadObj().GetMean()
-		elif 'incPathTime_' in allnames:
-                        pathname = '_'.join(allnames.split('_')[1:])
-                        if not pathname in names4:
-                                names4[pathname] = k.ReadObj().GetMean()
-
-        names2 = dict(sorted(six.iteritems(names1), key=operator.itemgetter(1),reverse=True)[:10])
-	names3 = sorted(names2, key=names2.get, reverse=True)
-        names5 = dict(sorted(six.iteritems(names4), key=operator.itemgetter(1),reverse=True)[:10])
-	names6 = sorted(names5, key=names5.get, reverse=True)
-
-        texfile = open(outfile+'-main.tex', 'w')
-        texfile.writelines(texpreamble)
-	if os.path.exists(infile.replace('.root','')+'-summary.txt'):
-                excludefile = open(infile.replace('.root','')+'-summary.txt', 'r')
-                texfile.write('\\newpage \section{Summary} \n \\begin{verbatim} \n')
-                for line in excludefile.readlines():
-                        texfile.write(line+'\n')
-                excludefile.close()
-		texfile.write('\end{verbatim}')
-        texfile.write('\\newpage \\chapter{10 Slowest Paths}\n')
-        texfile.write('\section{Average module (in path) time}\n')
-        for path in names3:
-                texfile.write('\\newpage \subsection{'+ path.replace('_','\_') +'} \centering \includegraphics[scale=0.35]{moduleInPathTimeSummary'+ path.replace('_','') +'temp.png}\n')
-                get_plot2(infile,'moduleInPathTimeSummary_'+path)
-        texfile.write('\section{Average module (in path) running time}\n')
-        for path in names3:
-                texfile.write('\\newpage \subsection{'+ path.replace('_','\_') +'} \centering \includegraphics[scale=0.35]{moduleInPathScaledTimeSummary'+ path.replace('_','') +'temp.png}\n')
-                get_plot2(infile,'moduleInPathScaledTimeSummary_'+path)
-        texfile.write('\section{Per event time for path}\n')
-        for path in names3:
-                texfile.write('\\newpage \subsection{'+ path.replace('_','\_') +'} \centering \includegraphics[scale=0.6]{pathTime'+ path.replace('_','') +'temp.png}\n')
-                get_plot1(infile,'pathTime_'+path)
-        texfile.write('\section{Per event incremental time for path}\n')
-        for path in names6:
-                texfile.write('\\newpage \subsection{'+ path.replace('_','\_') +'} \centering \includegraphics[scale=0.6]{incPathTime'+ path.replace('_','') +'temp.png}\n')
-                get_plot1(infile,'incPathTime_'+path)
-        texfile.write('\end{document}')
-        texfile.close()
-
-        texfile.close()
-
-	get_plot1(infile,'totalTime')
+    texpreamble = ['\documentclass[10pt,a5paper,landscape]{report}\n',
+                   '\\usepackage{graphicx}\n',
```

Comment:
I do not understand the changes here: this line starts with a double back-slash, but not the line above,
while both lines should produce LaTeX text lines starting with a single back-slash each.
??